scripts/replace.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+
+function copyFile(source, target, cb) {
+  let cbCalled = false;
+
+  const rd = fs.createReadStream(source);
+  rd.on('error', err => {
+    done(err);
+  });
+  const wr = fs.createWriteStream(target);
+  wr.on('error', err => {
+    done(err);
+  });
+  wr.on('close', () => {
+    done();
+  });
+  rd.pipe(wr);
+
+  function done(err) {
+    if (!cbCalled) {
+      cb(err);
+      cbCalled = true;
+    }
+  }
+}
+
+copyFile(
```

Comment:
This looks unfortunate and pretty brittle.  What led to needing to copy all these files?

Reply:
Hello @johnschoeman, 

This is for your information that we were working on the ticket SAF-390. Here we fixing GPS accuracy and reliability issues. We had to change in some java files of lib react-native-background-geolocation (that we are currently using). You see I changed in a node-module lib that we don't push. So somehow I need to keep the changes what I did in the lib.

For the same I created a script that will run after yarn or npm install to copy some files (where we changed) with existing node-modules lib react-native-background-geolocation in the machine from where we are creating the builds.

Please let me know if you have some other ways to get our changes in the node-modules lib. 

However Diarmid has tested these changes and given good feedback regarding fixes of GPS accuracy and reliability in android.

Reply:
got it, so we needed to overwrite a bunch of the geolocations libraries files because the base library is broken for our needs?  is it possible to fork the library instead or is there a patch available?

If we do need to copy n paste node module files in apost script, your script here is fine, its just unfortunate that we have to do it in the first place.

Reply:
Hello @johnschoeman, yes we needed to overwrite some files of the geolocation lib. Yes we can fork it. But for the first MVP, could we continue with the postscript and later we will fork it?